.github/workflows/ci.yml

```diff
@@ -0,0 +1,156 @@
+name: CI
+on:
+  push:
+    branch:
+      - master
+      - develop
+      - release**
+    tags:
+      - v**
+
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1.5'
+          - 'nightly'
+        os:
+          - ubuntu-latest
+        arch:
+          - x64
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup julia
+        uses: julia-actions/setup-julia@v1
+        with:
+          version: ${{ matrix.version }}
+          arch: ${{ matrix.arch }}
+      
+      - name: Cache Artifacts
+        uses: actions/cache@v1
+        env:
+          cache-name: cache-artifacts
+        with:
+          path: ~/.julia/artifacts
+          key: ${{ runner.os }}-test-${{ env.cache-name }}-${{ hashFiles('**/Project.toml') }}
+          restore-keys: |
+            ${{ runner.os }}-test-${{ env.cache-name }}-
+            ${{ runner.os }}-test-
+            ${{ runner.os }}-
+
+      - name: Julia Build Pkg
+        uses: julia-actions/julia-buildpkg@latest
+      
+      - name: Git Test Credentials
+        run: |
+          git config --global user.name Tester
+          git config --global user.email te@st.er
+      
+      - name: Run tests
+        uses: julia-actions/julia-runtest@latest
+        env:
+          IIF_TEST: false
```

Comment:
this guy should be for enable disabling test against branch in IIF `dfg_integration_test`